plugins/modules/xen_orchestra_instance.py

```diff
@@ -0,0 +1,335 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024 Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: xen_orchestra_instance
+short_description: Management of instances on Xen Orchestra
+description:
+  - Allows you to create/delete/restart/stop instances on Xen Orchestra.
+version_added: 10.3.0
+extends_documentation_fragment:
+  - community.general.attributes
+attributes:
+  check_mode:
+    support: none
+  diff_mode:
+    support: none
+options:
+  api_host:
+    description: API host to XOA API.
+    required: true
+    type: str
+  user:
+    description: Xen Orchestra user.
+    required: true
+    type: str
+  password:
+    description: Xen Orchestra password.
+    required: true
+    type: str
+  validate_certs:
+    description: Verify TLS certificate if using HTTPS.
+    type: bool
+    default: true
+  use_tls:
+    description: Use wss when connecting to the Xen Orchestra API.
```

Comment:
Arguably, this should be:
```suggestion
    description: Use C(wss) when connecting to the Xen Orchestra API.
```